kornia/utils/image.py

```diff
@@ -218,7 +218,7 @@ def _wrapper(input: Tensor, *args: Any, **kwargs: Any) -> Tensor:
         if not isinstance(input, Tensor):
             raise TypeError(f"Input input type is not a Tensor. Got {type(input)}")
 
-        if input.numel() == 0:
```

Comment:
the failing tests is because of it -- len(input.shape) didn't cover a `tensor([], size=(0, 1, 10, 20))`